docs/quick-start.md

```diff
@@ -190,6 +194,59 @@ $ kubectl logs test-695251588-cxwmc
 ...
 Hello World!
 ```
+### NATS
+
+If you have an exisiting NATS cluster, if pretty to easy to get started.
+
+Use the manifest to deploy Kubeless NATS triggers controller.
+
+```console
+kubectl create -f https://github.com/kubeless/kubeless/releases/download/$RELEASE/nats-$RELEASE.yaml
+```
+
+By default NATS trigger controller expects nats cluster is available as Kubernetes cluster service `nats.nats-io.svc.cluster.local:4222`. You can overide the default NATS cluster url used by setting the environment variable `NATS_URL` in the manifest. Once NATS trigger controller is setup you can deploy the function and associate function with a topic on the NATS cluster.
+
+```console
+$ kubeless function deploy pubsub-python-nats --runtime python2.7 \
+                                --handler test.foobar \
+                                --from-file test.py
+```
+
+After function is deployed you can use `kubeless trigger nats` CLI command to  associate function with a topic on NATS cluster as below.
+
+```console
+$ kubeless trigger nats create pubsub-python-nats --function-selector created-by=kubeless,function=pubsub-python-nats --trigger-topic test-topic
+```
+
+You could quickly test the functionality by publishing a message to the topic, and verifying that message is seen by the pod running the function.
+
+```console
+	kubeless trigger nats publish --url nats://nats-server-ip:4222 --topic test --message "Hello World!"
+```
+
+If you do not have NATS cluster its pretty easy to setup a NATS cluster. Run below command to deploy a [NATS operator](https://github.com/nats-io/nats-operator)
+
+```console
+$ kubectl apply -f https://raw.githubusercontent.com/nats-io/nats-operator/master/example/deployment-rbac.yaml
+```
+
+Once NATS operator is up and running run below command to deploy a NATS cluster
+
+```console
+echo '
+apiVersion: "nats.io/v1alpha2"
+kind: "NatsCluster"
+metadata:
+  name: "nats"
+spec:
+  size: 3
+  version: "1.1.0"
+' | kubectl apply -f - -n nats-io
+```
+
+Above command will create NATS cluster IP service `nats.nats-io.svc.cluster.local:4222` which is the default URL Kubeless NATS trigger contoller expects.
+
+At this point you are all set try Kubeless NATS triggers.
```

Comment:
I would continue the tutorial a little bit more and deploy a function using a NATS topic

Reply:
Expanded the quick start guide for NATS.